tests/python/relax/test_relax_tensor_ops.py

```diff
@@ -0,0 +1,259 @@
+
+    x = relax.Var("x", R.Tensor((2, 3), "float32"))
+    bb = relax.BlockBuilder()
+    with bb.function("main", [x]):
+        gv = bb.emit(relax.op.cos(x))
+        bb.emit_func_output(gv)
+
+    tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+
+def test_tanh():
+    @R.function
+    def expected(x: R.Tensor((2, 3), "float32")) -> R.Tensor((2, 3), "float32"):
+        gv: R.Tensor((2, 3), "float32") = R.tanh(x)
+        return gv
+
+    x = relax.Var("x", R.Tensor((2, 3), "float32"))
+    bb = relax.BlockBuilder()
+    with bb.function("main", [x]):
+        gv = bb.emit(relax.op.tanh(x))
+        bb.emit_func_output(gv)
+
+    tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+
+def test_sqrt():
+    @R.function
+    def expected(x: R.Tensor((2, 3), "float32")) -> R.Tensor((2, 3), "float32"):
+        gv: R.Tensor((2, 3), "float32") = R.sqrt(x)
+        return gv
+
+    x = relax.Var("x", R.Tensor((2, 3), "float32"))
+    bb = relax.BlockBuilder()
+    with bb.function("main", [x]):
+        gv = bb.emit(relax.op.sqrt(x))
+        bb.emit_func_output(gv)
+
+    tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+
+def test_log():
+    @R.function
+    def expected(x: R.Tensor((2, 3), "float32")) -> R.Tensor((2, 3), "float32"):
+        gv: R.Tensor((2, 3), "float32") = R.log(x)
+        return gv
+
+    x = relax.Var("x", R.Tensor((2, 3), "float32"))
+    bb = relax.BlockBuilder()
+    with bb.function("main", [x]):
+        gv = bb.emit(relax.op.log(x))
+        bb.emit_func_output(gv)
+
+    tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+
+def test_sigmoid():
+    @R.function
+    def expected(x: R.Tensor((2, 3), "float32")) -> R.Tensor((2, 3), "float32"):
+        gv: R.Tensor((2, 3), "float32") = R.sigmoid(x)
+        return gv
+
+    x = relax.Var("x", R.Tensor((2, 3), "float32"))
+    bb = relax.BlockBuilder()
+    with bb.function("main", [x]):
+        gv = bb.emit(relax.op.sigmoid(x))
+        bb.emit_func_output(gv)
+
+    tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+
+def test_less():
+    @R.function
+    def expected(
+        x: R.Tensor((2, 3), "float32"), y: R.Tensor((2, 1), "float32")
+    ) -> R.Tensor((2, 3), "bool"):
+        gv: R.Tensor((2, 3), "bool") = R.less(x, y)
+        return gv
+
+    x = relax.Var("x", R.Tensor((2, 3), "float32"))
+    y = relax.Var("y", R.Tensor((2, 1), "float32"))
+    bb = relax.BlockBuilder()
+    with bb.function("main", [x, y]):
+        gv = bb.emit(relax.op.less(x, y))
+        bb.emit_func_output(gv)
+
+    tvm.ir.assert_structural_equal(bb.get()["main"], expected)
+
+
+def test_ewise_fma():
+    @R.function
+    def expected(
+        x: R.Tensor((2, 3, 4), dtype="float32"),
+        y: R.Tensor((2, 3, 4), dtype="float32"),
+        z: R.Tensor((2, 3, 4), dtype="float32"),
+    ) -> R.Tensor((2, 3, 4), dtype="float32"):
+        gv: R.Tensor((2, 3, 4), dtype="float32") = R.ewise_fma(x, y, z)
+        return gv
```

Comment:
Please add tetstcases that focuses on deduction

Including:
- symbolic shape deduction
- cases that can trigger error (in that case only block builder is fine) reporting.
- cases that will results in an erased outcome

We do not need to cover every op, but needs to do so for each kind, like binrary broadcast, and binary broadcast binary.

Consider follow the auxiliary function here https://github.com/mlc-ai/relax/blob/structinfo/tests/python/relax/test_analysis_struct_info_analysis.py#L335

Given the number of operators might grow, we can consider split the file

- test_op_unary.py
- test_op_binary.py